tests/storage/cdi_import/test_import_http.py

```diff
@@ -539,7 +539,7 @@ def test_blank_disk_import_validate_status(data_volume_multi_storage_scope_funct
                 "dv_name": "cnv-3065",
                 "file_name": Images.Cdi.QCOW2_IMG,
                 "source": HTTPS,
-                "size": "100Mi",
```

Comment:
What was the problem with 100Mi size? Which storage class do you use?
If DV is not getting created because the storage request is too small, then it's a bug.
An annotation should be added to the `StorageProfile`, introduced here: https://github.com/kubevirt/containerized-data-importer/pull/3711